poc_mobile_verifier/src/heartbeats.rs

```diff
@@ -1,6 +1,6 @@
 //! Heartbeat storage
 
-use crate::{cell_type::CellType, Error, Result};
+use crate::cell_type::CellType;
```

Comment:
curious to understand why 'anyhow' is in use on some modules and not others ? What is the decision making around whether to use it or default to standard rust Result type

Reply:
The plan is to convert every crate over to use anyhow, but I figured one PR per module would be easier for people to digest

Reply:
i.e. every _top level application_ will use the anyhow crate to make errors have more context. Any library behavior will use a specific Error subtype

Reply:
Jeff is correct, i was referring to why only some 'modules' are using anyhow and others are continuing to use the prior style